scapy/layers/dhcp.py

```diff
@@ -220,6 +220,7 @@ def getfield(self, pkt, s):
     150: IPField("tftp_server_address", "0.0.0.0"),
     159: "v4-portparams",
     160: StrField("v4-captive-portal", ""),
+    161: "mud-url",
```

Comment:
Shouldn't you use a `StrField` ? Pretty sure that's why your test didn't pass

Reply:
Thanks for pointing that out! This problem should now be fixed :)